lib/pages/image_selector/image_selector.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+
+class ImageSelector extends StatefulWidget {
+  const ImageSelector({Key? key}) : super(key: key);
```

Comment:
Perhaps we can change this to its abbreviated state? 

`const ImageSelector({super.key});`